x-pack/plugin/apm-integration/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+grant {
+    permission java.lang.RuntimePermission "accessClassInPackage.sun.misc";
+    permission java.lang.reflect.ReflectPermission "suppressAccessChecks";
+    permission java.lang.RuntimePermission "accessDeclaredMembers";
+    // required by io.grpc.internal.DnsNameResolver in grpc-core
+    permission java.net.NetPermission "getProxySelector";
```

Comment:
```suggestion
    permission java.net.NetPermission "getProxySelector";
    permission java.lang.RuntimePermission "getClassLoader";
    permission java.net.SocketPermission "*", "connect,resolve";
```
With these two permissions, I am able to start ES and start sending data to the cloud APM server. Somehow they are not needed by `ApmIT`, but are necessary when actually running the node.

Reply:
> With these two permissions, I am able to start ES and start sending data to the cloud APM server. Somehow they are not needed by `ApmIT`, but are necessary when actually running the node.

Thanks for catching this Yang, I did not have the time yesterday to get that far :)